src/app/ReadClient.h

```diff
@@ -612,6 +644,11 @@ class ReadClient : public Messaging::ExchangeDelegate
 
     System::Clock::Timeout mLivenessTimeoutOverride = System::Clock::kZero;
 
+    // When the liveness timeout fired after `mPeerActiveUntilTimestamp`, the ReadClient will enter "IdleSubscription" state
+    // until WakeUp() is called. The Zero value means the device should always be active.
+    System::Clock::Timeout mPeerActivePeriod           = System::Clock::kZero;
+    System::Clock::Timestamp mPeerActiveUntilTimestamp = System::Clock::kZero;
```

Comment:
Comment doesn't really explains what the members represent imo.

Reply:
updated, thanks